book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,288 @@
+  apply if the tests in question provide substantial value via early detection
+  of critical problems.
+- Building the target in CI must not take substantially longer than the current
+  slowest target in CI, and should not substantially raise the maintenance
+  burden of the CI infrastructure. This requirement is subjective, to be
+  evaluated by the devops team, and will take the community importance
+  of the target into account.
+- Tier 2 targets should, if at all possible, support cross-compiling. Tier 2
+  targets should not require using the target as the host for builds, even if
+  the target supports host tools.
+- Tier 2 targets must not impose burden on the authors of pull requests, or
+  other developers in the community, to ensure that tests pass for the target.
+  In particular, do not post comments (automated or manual) on a PR that derail
+  or suggest a block on the PR based on tests failing for the target. Do not
+  send automated messages or notifications (via any medium, including via `@`)
+  to a PR author or others involved with a PR regarding the PR breaking tests
+  on a tier 2 target, unless they have opted into such messages.
+- The target maintainers should regularly run the testsuite for the target, and
+  should fix any test failures in a reasonably timely fashion.
+- All requirements for tier 3 apply.
+
+A tier 2 target may be demoted or removed if it no longer meets these
+requirements. Any proposal for demotion or removal will be CCed to the target
+maintainers, and will be communicated widely to the Zebra community before being
+dropped from a stable release. (The amount of time between such communication
+and the next stable release may depend on the nature and severity of the failed
+requirement, the timing of its discovery, whether the target has been part of a
+stable release yet, and whether the demotion or removal can be a planned and
+scheduled action.)
+
+## Tier 1 target policy
+
+At this tier, the Zebra project guarantees that a target builds and passes all
+tests, and will reject patches that fail to build or pass the testsuite on a
+target. We hold tier 1 targets to our highest standard of requirements.
+
+A proposed new tier 1 target must be reviewed and approved by the core
+engineering team based on these requirements. In addition, the release team must
+approve the viability and value of supporting the target. For a tier 1 target,
+this will typically take place via a full RFC proposing the target, to be
+jointly reviewed and approved by the core engineering team and release team.
+
+In addition, the devops team must approve the integration of the target
+into Continuous Integration (CI), and the tier 1 CI-related requirements. This
+review and approval may take place in a PR adding the target to CI, by an
+devops team member reporting the outcome of a team discussion, or by
+including the devops team in the RFC proposing the target.
```

Comment:
We don't use RFCs like this for Zebra, can you remove all the references to RFCs in this document?

Reply:
Fixed by [docs(release): remove references to use RFCs](https://github.com/ZcashFoundation/zebra/pull/4773/commits/641a6d08a379dc64ae0b1e8128649056542c1f23)